core/sup_tree_core/executor_pool/memcache_manager.ex

```diff
@@ -0,0 +1,103 @@
+# Copyright(c) 2015-2018 ACCESS CO., LTD. All rights reserved.
+
+use Croma
+
+defmodule AntikytheraCore.ExecutorPool.MemcacheManager do
+  @moduledoc """
+  A `GenServer` to manage memcache for each executor pool.
+  """
+
+  @timeout         300_000
+  @max_records_num 10
+  @max_record_size 65536
```

Comment:
To make it a bit more accurate let's rename `@max_record_size` to `@max_value_size`.

Reply:
The limit was removed.